src/pages/permission-management/user/roleAuth/Index.js

```diff
@@ -0,0 +1,144 @@
+import React, { PureComponent, Fragment } from 'react';
+import { connect } from 'dva';
+import router from 'umi/router';
+import {
+  List,
+  Card,
+  Row,
+  Col,
+  Radio,
+  Input,
+  InputNumber,
+  Progress,
+  Button,
+  Icon,
+  Dropdown,
+  Menu,
+  Avatar,
+  Modal,
+  Form,
+  DatePicker,
+  Select,
+  message,
+  Transfer,
+  Divider,
+} from 'antd';
+import StandardTable from '@/components/StandardTable';
+import PageHeaderWrapper from '@/components/PageHeaderWrapper';
+import styles from '../../../List/TableList.less';
+
+const FormItem = Form.Item;
+const { Option } = Select;
+
+@connect(({ user, role, loading }) => ({
+  user,
+  role,
+  loading: loading.models.user,
+}))
+@Form.create()
+class UserRoleAuth extends PureComponent {
+
+  state = {
```

Comment:
Codacy found an issue: [State initialization should be in a constructor](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=6006324)